pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/auth_proxy_client_role.go

```diff
@@ -43,6 +44,11 @@ func (f *AuthProxyClientRole) SetTemplateDefaults() error {
 const clientClusterRoleTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: metrics-reader
+    app.kubernetes.io/component: auth-proxy
```

Comment:
```suggestion
    app.kubernetes.io/component: metrics
```

Here we apply a manifest that is for the metrics component as well. We do not use this one if we do not enable the metrics

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.